### PR TITLE
AIT-3 docs (wip)

### PR DESCRIPTION
**Pull request by rajkaramchedu**
Ongoing AIT-3 docs. This PR will be updated with corrections, additions until we are ready to bring the docs live. Please comment freely. 

- **TODO**: Documenting the CLI commands and the flow for the owner and the operator. @sherry-x, can I use this documentation to start with? https://www.notion.so/aptoslabs/AIT3-Testing-Story-bd267627981a4520ad2fb68fd961d59c
- Also, @sherry-x, the above Notion doc belongs in the [Connecting to the AIT and Joining and Leaving the Validator Set ](https://deploy-preview-2513--aptos-developer-docs.netlify.app/nodes/ait/connect-to-testnet)docs, correct? 
- The above Notion doc pretty much covers everything we need from your this [PR](https://github.com/aptos-labs/aptos-core/pull/2867), right, @igor-aptos?
- @sherry-x or anyone else: Anything else we should cover in these docs? 
- **Questions**: 
- 1. Is the operator public key same as the `validator_network_public_key` from the "aptosbot.yaml" file for the validator? 
- 2. In which step does the owner transfer coins to the operator and voter wallets for gas purposes? 
- 3. In the Aptos Governance/proposals/staking UI section, we are providing the wallet address of the operator and voter. Does it mean that the wallet address of the operator is the same as the operator address on the blockchain? 
- 4. I don't think I really understood where exactly is the match made between the selected node operator and the operator wallet address? When I create the operator wallet, that address is surely different from the operator public key I provided when registering with Aptos CP, isn't it? Some behind the scenes correlation is happening and I think I should make that explicit in my docs. Can anyone help? Perhaps @CapCap?
Thanks! 

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/2513)

### CODE REVIEW

File: developer-docs-site/docs/nodes/ait/whats-new-in-ait3.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+# Whats New in AIT-3
+
+:::caution DRAFT-only
+These AIT-3 docs are draft-only for now.
+:::
+
+Several new features are up for testing by the AIT-3 participants. See below:
+
+### Aptos Wallet
```
Comment (sherry-x) [inline developer-docs-site/docs/nodes/ait/whats-new-in-ait3.md:17]: Petra! we can refer to its new name now :D

File: developer-docs-site/docs/nodes/ait/whats-new-in-ait3.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+- The new Aptos Wallet, available as a Chrome webapp extension. You will use this Wallet to participate in staking and governance in AIT-3.
+
+### Owner, operator and voter personas
+
+- Participation in staking and governance is now enabled with three personas: an owner, an operator and a voter.  
+- The owner is the owner of the funds, typically an investor. For example, the owner Bob can assign his operator address to the account of Alice, a trusted validator operator. See [How a custodian can stake on Aptos](/nodes/staking#how-a-custodian-can-stake-on-aptos) for more details on owner and operator definitions.
```
Comment (sherry-x) [inline developer-docs-site/docs/nodes/ait/whats-new-in-ait3.md:24]: Let's remove the "typically an investor" part

File: developer-docs-site/docs/nodes/ait/whats-new-in-ait3.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+- Participation in staking and governance is now enabled with three personas: an owner, an operator and a voter.  
+- The owner is the owner of the funds, typically an investor. For example, the owner Bob can assign his operator address to the account of Alice, a trusted validator operator. See [How a custodian can stake on Aptos](/nodes/staking#how-a-custodian-can-stake-on-aptos) for more details on owner and operator definitions.  
+- Voter is a new persona being introduced in AIT-3. An owner can designate a voter. This enables the voter to participate in governance.
+
+### Staking
+
+- A new staking UI, making it easier to manage staking.
+<!--- TODO: Is rotating the keys supported in AIT-3? --->
+- Rotating the keys.
```
Comment (sherry-x) [inline developer-docs-site/docs/nodes/ait/whats-new-in-ait3.md:31]: rotating key is actually part of node operation, let's remove from here

File: developer-docs-site/docs/nodes/ait/whats-new-in-ait3.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+- Off-chain upgrades, such as:
+  - Changes to consensus.
+  - Upgrade the Move VM version.
+  - See the version of the software.
+- Nodes
+  - Nodes dynamically joining and leaving when the network is under load. Require the node to leave the network for at least X minutes duration.
+  - Send all types of transactions to the Aptos blockchain to test for a consistent load on the network and monitor the cost.
+  - Operator to rollback the software from version B to version A, for testing.
+  - Operator to update node configuration, for testing.
+  - Operator to restore the node from the backup data.
```
Comment (sherry-x) [inline developer-docs-site/docs/nodes/ait/whats-new-in-ait3.md:55]: remove those details. replace with:

Community to vote on proposals. Proposals will be in the following areas:
- Blockchain on-chain configuration, e.g. Epoch length
- Staking configuration, e.g. reward rate
- Governance configuration, e.g. voting power requirement
- Move module upgrades

File: developer-docs-site/docs/nodes/ait/whats-new-in-ait3.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+To participate in testing the staking and the governance features in the AIT-3, you will create three personas. See below an explanation of these personas: 
+
+- **Owner**: The owner account contains the validator settings and the coins. The coins are airdropped into the owner account.
+- **Operator**: If you are the owner, then, using your owner key, you will select the specific operator and you will:
+  - Manage the settings for the specific validator, and
+  - Delegate the stake pool to the validator.
+  - The operator public key is same key you (owner) used while registering your validator node with the Aptos Community Platform. This is the  `validator_network_public_key` from the "aptosbot.yaml" file for the validator.
+- **Voter**: You will use the voter key to sign the governance votes in the transactions.
+
```
Comment (sherry-x) [inline developer-docs-site/docs/nodes/ait/whats-new-in-ait3.md:67]: This feels like a duplication of the "Owner, operator and voter personas" section above

**Review by sherry-x — APPROVED** [review]
I will probably need to make some tweak later once I added the operation details, but this is great starting point!

**Comment by rajkaramchedu** [comment]
Governance questions for @movekevin. 

Hey Kevin - Hey, I am now writing up the governance doc for the AIT-3. I am using your [gdoc](https://docs.google.com/document/d/1acY6-BrrP3V47sYSPzUuyDK2zt8XQCG-kbKOzr-HI_w/edit#) as my reference. [This is my first draft](https://deploy-preview-2513--aptos-developer-docs.netlify.app/concepts/governance).  I want to ask you a bunch of questions on this topic. Can you please address them? Thanks!

Questions
---------
1. Is the picture I drew up accurate and complete in the draft?
2. In the voting period, do have anything like a quorum requirement? I mean after the voting period is over, if less than quorum number of votes are cast, then the proposal is not ready to be resolved. Do we have anything like this already and I missed it?
3. “The proposer’s backing stake pool needs to have the minimum proposer stake required" in https://github.com/aptos-labs/aptos-core/blob/27a255ebc662817944435349afc4ec33ea317e64/aptos-move/framework/aptos-framework/sources/aptos_governance.move#L7
   1. How is “backing stake pool” defined? Is this the same as the staked amount by the validator, or a subset of it or ?
   2. Who sets the minimum proposer stake required? Is this the same as the “required bond amount” below?
4. “The proposer’s stake needs to be at least the required bond amount.” in https://github.com/aptos-labs/aptos-core/blob/27a255ebc662817944435349afc4ec33ea317e64/aptos-move/framework/aptos-framework/sources/aptos_governance.move#L193 How is the required bond amount set and by whom? By Aptos team?
5. In the Voting Module section of your gdoc, the item 3 says, “A proposer, through the governance module, can call Voting::create_proposal to create a proposal. Create_proposal cannot be called directly not through the governance module.” But elsewhere in the gdoc it also says, “Proposers can create a proposal by calling `AptosGovernance::create_proposal`”. So are these two ways to create a proposal?
6. Proposer vs voter needs a bit more clarity for me. Both the voter and a proposer can create a proposal and can vote. So is there any distinction between a “proposer” and a “voter”? Do you intend these terms to be strict (like how a “validator” is someone who runs a validator node)? If so, what distinguishes one from the other?
7. I am having a hard time understanding the whole notion of resolution script and the hash and why we have the resolution mechanism this way. I am just trying to get at the motivation for this mechanism. But I think that is only because I haven’t spent enough time reading through your code. Any chance you have an example code for a proposal that shows the resolution script and how a resolver runs it and what happens then? This might even address my below question.
8. I need to show for the AIT-3 docs an example governance flow and describe what we are expecting from the specific personas: owner, operator and the voter. Do you have any pointers for me here?
Thanks!

**Pull request merged**